src/applications/dhp-connected-devices/components/UnauthenticatedPageContent.jsx

```diff
@@ -16,7 +16,7 @@ export const UnauthenticatedPageContent = () => {
           If you don’t have any of these accounts, you can create a free ID.me
           account now.
         </div>
-        <div className="button">
+        <div className="dhp-button">
```

Comment:
A more descriptive class name could be `dhp-login-button` but up to you if you want to change it.

Reply:
@yonidoronpeters good catch on the photos being flipped! And I went ahead and changed the class name, good call there. Thanks!